<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionTypeWidget</class>
 <widget class="QWidget" name="SessionTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QPushButton#btnConnect{background-color:rgb(80,180,80)}
QWidget{background-color: rgba(0,0,0,0);color:white;border-radius:5px}
QWidget#mainWidget{background-color:rgba(128,128,128,25%);}
QWidget#frmProfile,QWidget#pageProfile{background-color:rgba(108,8,156,25%);}
QPlainTextEdit,QLineEdit,QListWidget,QComboBox,QTextEdit,QSpinBox{background-color: rgba(255,255,255,50%); color: black;}

QTableWidget{
background-color: rgba(0,0,0,50%);
color: white;
border: 1px solid rgba(255,255,255,50%);
}

QLabel#lblLastOnline{color:lightblue;}
QLabel#lblError,QLabel#lblCamMissing,QLabel#lblAudioMissing{color:red; background-color:rgba(255,0,0,30%);}

QLineEdit:!enabled, QListWidget:!enabled,QComboBox:!enabled,QTextEdit:!enabled,QPlainTextEdit:!enabled{background-color: rgba(0,0,0,30%);color:white;border:0}
QDateEdit::down-arrow:!enabled,QTimeEdit::down-arrow:!enabled,QDateEdit::up-arrow:!enabled,QTimeEdit::up-arrow:!enabled,QComboBox::drop-down:!enabled{background-color:rgba(0,0,0,0%);}

QTextEdit#txtProfile{background-color: rgba(255,255,255,80%);color:black;border:0}
QTextEdit:!enabled#txtProfile{background-color: rgba(255,255,255,60%);color:black;border:0}

QFrame#frmSelUser{background-color:rgba(100,100,200,50%);}
QFrame#frmAudio,QFrame#frmCam1,QFrame#frmCam2,QFrame#frmSensors,QFrame#frmWebRTC,QFrame#frmVirtualCam,QFrame#frmExternalPrograms{background-color:rgba(100,100,200,35%);}

QCheckBox::indicator:unchecked:!enabled,QListWidget::indicator:unchecked:!enabled { background-color:rgba(128,0,0,100%);}
QCheckBox::indicator:checked:!enabled,QListWidget::indicator:checked:!enabled { background-color:green;}
QCheckBox::indicator:unchecked,QListWidget::indicator:unchecked{ background-color:red;border-radius:3px;}
QCheckBox::indicator:checked, QListWidget::indicator:checked {background-color:rgb(0,255,0);border-radius:3px;}

QPushButton{background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2198c0, stop: 1 #0d5ca6);}
QPushButton:checked{background-color:rgb(128,128,128); border:1px solid rgba(255,255,255,50%);}
QPushButton:hover,QPushButton:checked{background-color:  rgba(255,255,255,75%);color:black;}
QPushButton#btnEditUser:hover{background-color:  rgba(255,255,255,75%);color:black;}
QPushButton#btnEdit,QPushButton#btnDelete,QPushButton#btnExisting{background-color:  rgba(255,255,255,0%);border:0 px;}
QPushButton#btnEdit:hover,QPushButton#btnDelete:hover,QPushButton#btnExisting:hover{background-color:  rgba(255,255,255,25%);border:0 px;}
QPushButton#btnExisting:checked{background-color:rgba(0,100,200,50%);}
QPushButton#btnUpgrade{background-color:rgba(200,0,0,90%);}

QToolButton:hover{background-color:  rgba(255,255,255,50);}

QTreeWidget::branch:closed:has-children:has-siblings  {
            border-image: none;
            image: url(:/pictures/controls/branch_closed.png);
    }
     
    QTreeView::branch:open:has-children:!has-siblings,
    QTreeView::branch:open:has-children:has-siblings   {
            border-image: none;
            image:url(:/pictures/controls/branch_opened.png);
    }

QWidget#lineOnline{background-color:blue;}
QWidget#lineOffline{background-color:red;}
QWidget#lineBoth{background-color:rgb(255,0,255);}


QTabWidget::pane { 
	background-color: rgba(128,128,140,25%);
	border-radius: 5px;
    border: 2px solid rgba(128,128,140,50%);
 }

 QTabWidget::tab-bar {
     left: 5px; 
 }


 QTabBar::tab,QTableWidget::tab,QHeaderView::section {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgba(128,128,140,50%),  stop: 1.0  rgba(128,128,140,25%));
     border: 2px solid  rgba(128,128,140,50%);
     border-bottom: 0px;
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     padding: 2px;
min-height:25px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0  rgba(128,128,140,70%),  stop: 1.0  rgba(128,128,140,50%));
 }

 QTabBar::tab:selected {
     border-color:rgba(128,128,140,60%);
     border-bottom-color:rgba(128,128,140,60%);
 }

 QTabBar::tab:!selected {
     margin-top: 2px; 
 }
*/

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="icoTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../resources/TeraClient.qrc">:/icons/session_type.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Type de séance</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabNav">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabDashboard">
      <attribute name="icon">
       <iconset resource="../../resources/TeraClient.qrc">
        <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Informations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Éditer</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/TeraClient.qrc">
             <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TeraForm" name="wdgSessionType" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameButtons">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Sauvegarder</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/TeraClient.qrc">
              <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUndo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Annuler</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/TeraClient.qrc">
              <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProjects">
      <attribute name="icon">
       <iconset resource="../../resources/TeraClient.qrc">
        <normaloff>:/icons/site-icon.png</normaloff>:/icons/site-icon.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Sites / Projets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTreeWidget" name="treeProjects">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnProjects">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Mettre à jour les sites / projets associés</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/TeraClient.qrc">
           <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConfig">
      <attribute name="icon">
       <iconset resource="../../resources/TeraClient.qrc">
        <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Paramètres</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="TeraForm" name="wdgSessionTypeConfig" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUpdateConfig">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Mettre à jour la configuration</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/TeraClient.qrc">
           <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TeraForm</class>
   <extends>QWidget</extends>
   <header>TeraForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/TeraClient.qrc"/>
 </resources>
 <connections/>
</ui>
